docs/data/date-pickers/shortcuts/CustomizedRangeShortcuts.tsx

```diff
@@ -85,12 +101,20 @@ function CustomRangeShortcuts(props: PickersShortcutsProps<DateRange<Dayjs>>) {
         dense
         sx={(theme) => ({
           display: 'flex',
+          maxWidth: '625px', // DAY_CALENDAR_FOR_RANGE_WIDTH * 2 + 1 (margin)
           px: theme.spacing(4),
           '& .MuiListItem-root': {
             pt: 0,
             pl: 0,
             pr: theme.spacing(1),
           },
+          scrollbarGutter: 'stable',
+          overflow: 'auto',
+          '::-webkit-scrollbar': {
+            display: 'none',
+          },
+          MsOverflowStyle: 'none',
+          scrollbarWidth: 'none',
```

Comment:
I wasn't able to avoid the layout shift when the horizontal scrollbar is shown on desktop. 🙈 
WDYT about a hybrid approach to facilitate a more proper mobile behavior?
```suggestion
          overflowX: 'hidden',
          '@media (pointer: fine)': {
            overflowX: 'auto',
            MsOverflowStyle: 'none',
            scrollbarWidth: 'none',
          },
          '@media (pointer: none), (pointer: coarse)': {
            overflowX: 'auto',
          },
```